ui/app/mirrors/create/mirrorcards.tsx

```diff
@@ -58,7 +58,7 @@ const MirrorCards = ({
                 borderRadius: '1rem',
               }}
             >
-              <div>
+              <label style={{ display: 'block' }}>
```

Comment:
Experiment: does this let me click mirror type box without having to click on content / radio button in box?